src/ng_link/scripts/dispim_link.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+
+from ng_link import NgState
+
+import xml_parsing
+import link_utils
+
+def apply_deskewing(matrix_3x4: np.ndarray, theta: float = 45) -> np.ndarray:
```

Comment:
I believe the documentation is super important for the functions and the script itself. I encourage you to use the interrogate package to validate documentation